install/kubernetes/helm/istio/charts/kiali/templates/deployment.yaml

```diff
@@ -50,21 +50,7 @@ spec:
               name: {{ .Values.dashboard.secretName }}
               key: {{ .Values.dashboard.passphraseKey }}
         - name: PROMETHEUS_SERVICE_URL
-          value: http://prometheus:9090
-{{- if .Values.dashboard.grafanaURL }}
```

Comment:
So kiali deployment will read those values from configmap by default?

Reply:
@gyliu513 
are you referring to Grafana/Jaeger values or PROMETHEUS_SERVICE_URL?

Reply:
@gbaufake Yes, because I found that you are now removing those parameters from Kiali deployment and moving them to configmap, so I want to know how Kiali deployment can get those parameters from configmap?

Reply:
@gyliu513  The grafana/jaeger are now being handled automatic on Kiali (https://github.com/kiali/kiali/blob/v0.7/config/config.go#L158-L161)

Reply:
Good to know, thanks @gbaufake